app/assets/sass/utilities/_grid.scss

```diff
@@ -0,0 +1,16 @@
+.app-\!-width-one-fifth {
```

Comment:
Where is this used?

Reply:
Both `app-!-width-one-fifth` and `app-!-width-one-sixth` are not used currently/anymore. Happy to remove?